### ISSUE TITLE
Version 1.3.6 did not reset EEPROM

### ISSUE THREAD

**Issue by aweussom** [NuclearPhoenixx/Kobra_Neo#7]
Previous versions did.

I think it is a good idea to clear eeprom when installing new firmware?

**Comment by NuclearPhoenixx** [comment]
Nothing changed in that regard. I literally only uncommented the single line that brought back the babystepping menu and enabled it in the config as you saw.

Why do you think this is a good idea in general? I for myself don't want revert a whole bunch of settings after each tiny update. I totally get where you're coming from, but I wouldn't know why it's necessary here. If you want to be 100% on the safe side, you can always reset it manually. You don't need to with this kind of tiny update that just changes some of the GUI.

**Comment by NuclearPhoenixx** [comment]
Yup, I added a note with bc15ef2e45fb0e6006018e813145d0dbd47438fb.

No idea when or how it resets the EEPROM, except when there are errors. I have had it reset on itself like once or twice after updates, but most of the time it's fine. A lot of other users in general have reported issues with things like the Z offset randomly not saving on the Neo. So my best bet would be Anycubic did something to the saves, or maybe the Marlin version they chose was a bit buggy at that time.

At least with some of the forks (including this one) you can nuke the EEPROM in the advanced settings.

**Comment by aweussom** [comment]
Yes, reloading defaults works fine.
